app/renderer/js/pages/preference/general-section.ts

```diff
@@ -418,6 +447,30 @@ export default class GeneralSection extends BaseSection {
 		});
 	}
 
+	setLocale(): void {
+		const langMenu = document.querySelector('#app-language .lang-menu');
+		const langList: any = document.querySelector('.lang-menu');
+		/*
+			The next three lines set the selected language visible on the dropdown button always
+			eg if user selects Español, then he shouldn't see Bulgarian when dropdown isn't opened
+			as it is the first option in the menu
+		*/
+		let localeIndex = ConfigUtil.getConfigItem('localeIndex');
+		localeIndex = (localeIndex === null) ? 5 : localeIndex;
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+		(langList.options[localeIndex] as any).setAttribute('selected', '');
+		langMenu.addEventListener('change', () => {
+			let opt;
+			for (opt of langList.options) {
+				if (opt.selected) {
+					break;
+				}
+			}
+			ConfigUtil.setConfigItem('appLocale', opt.value);
```

Comment:
use `event.target.value` here and `event.target["selectedIndex"]` below instead.

Reply:
Can use as following for TS errors - 
```js
(event.target as HTMLSelectElement).value;
```